src/common/settings.h

```diff
@@ -482,21 +485,22 @@ struct Values {
     SwitchableSetting<TextureFilter> texture_filter{TextureFilter::None, "texture_filter"};
     SwitchableSetting<TextureSampling> texture_sampling{TextureSampling::GameControlled,
                                                         "texture_sampling"};
-
     SwitchableSetting<LayoutOption> layout_option{LayoutOption::Default, "layout_option"};
     SwitchableSetting<bool> swap_screen{false, "swap_screen"};
     SwitchableSetting<bool> upright_screen{false, "upright_screen"};
     SwitchableSetting<float, true> large_screen_proportion{4.f, 1.f, 16.f,
                                                            "large_screen_proportion"};
+    // I think the custom_layout setting below is no longer needed
+    // since custom layout is now just part of the layout option above?
     Setting<bool> custom_layout{false, "custom_layout"};
     Setting<u16> custom_top_x{0, "custom_top_x"};
     Setting<u16> custom_top_y{0, "custom_top_y"};
-    Setting<u16> custom_top_width{400, "custom_top_width"};
-    Setting<u16> custom_top_height{240, "custom_top_height"};
-    Setting<u16> custom_bottom_x{40, "custom_bottom_x"};
-    Setting<u16> custom_bottom_y{240, "custom_bottom_y"};
-    Setting<u16> custom_bottom_width{320, "custom_bottom_width"};
-    Setting<u16> custom_bottom_height{240, "custom_bottom_height"};
+    Setting<u16> custom_top_width{800, "custom_top_width"};
+    Setting<u16> custom_top_height{480, "custom_top_height"};
+    Setting<u16> custom_bottom_x{80, "custom_bottom_x"};
+    Setting<u16> custom_bottom_y{500, "custom_bottom_y"};
+    Setting<u16> custom_bottom_width{640, "custom_bottom_width"};
+    Setting<u16> custom_bottom_height{480, "custom_bottom_height"};
```

Comment:
If they are no longer needed, remove them.

Reply:
Some users might still have them in their INI, and I'm not absolutely certain there isn't backup code to honor them somewhere. I could look into it more.

Reply:
Please do

Reply:
It is used in, as far as I can tell, three places as an alternative legacy way of using a Custom Layout. It could be easily removed - the only result would be that people currently using old .ini files to get a custom layout will no longer get their custom layout without re-enabling it through the UI.

Reply:
Please let me know if you want me to go forward with removing this setting and its references.

Reply:
This legacy coordinate system has already been removed elsewhere, so it would be appropriate to remove it here as well.

Reply:
The only one that can be removed is the first one (custom_layout boolean). All of the others are still used by the framebuffer to build the layout. I will do that tonight or tomorrow, weekend latest.